src/neptune/api.d

```diff
@@ -0,0 +1,166 @@
+/*******************************************************************************
+
+    helper methods to access the octod/github API
+
+    Copyright:
+        Copyright (c) 2017 Sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module neptune.api;
```

Comment:
It may make sense to keep neptine package for most generic bits applicable to all tools, i.e. move existing `semver.Version` to `neptune.Version`